publish.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Good point @t8y8 though I'm hesitant to take a dependency on an external module just for making our job of pushing to pypi easier.
Eventually, I'd like to automate publishing to pypi when we create the release tag.